db/migrate/20151123010203_initial_db.rb

```diff
@@ -8,79 +8,74 @@ def self.up
   end
 
   def self.create_categories
-    unless ActiveRecord::Base.connection.table_exists?('categories')
-      create_table 'categories' do |t|
-        t.string 'name', limit: 255, null: false
-      end
+    return if ActiveRecord::Base.connection.table_exists?('categories')
+    create_table 'categories' do |t|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 't' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->